distributions/standalone/src/main/resources/MIGRATION.txt

```diff
@@ -7,6 +7,7 @@ Please check https://developer.mulesoft.com/docs/display/current/Mule+ESB for mo
 Migration changes from 3.8.0 to 3.8.x
 MULE-10100: Processing after a synchronous until successful resulting in a VoidMuleEvent will now continue with the original event.
 MULE-10686: XML entity expansion in Jersey is now disabled by default because it allows DoS attacks. To restore previous behaviour use the mule.xml.expandInternalEntities="true" property.
+MULE-10306: XML entity expansion is now disabled by default because it allows DoS attacks. To restore previous behaviour use the mule.xml.expandInternalEntities="true" property.
```

Comment:
are the double quotes necessary? how would I enabled it from the console or add a wrapper.conf entry?